code-challenges.js

```diff
@@ -7,38 +7,73 @@
+// Then I'll repeat this same step slichtly changing it to fit the rest of the problems in this question.
+// When I log these problems I am expecting to get back the results that are expected of me in the question.
+
+// const temp1 = 42
+// const temp2 = 350
+// const temp3 = 212
+//
+// const boilingPoint = (currentTempt) => {
+//   if(currentTempt < 212){
+//     return `${currentTempt} is below boiling point`
+//   }else if (currentTempt > 212){
+//     return `${currentTempt} is above boiling point`
+//   }else if (currentTempt = 212){
+//     return `${currentTempt} is at boiling point`
+//   }
+// }
+// console.log(boilingPoint(temp1)) 42 is below boiling point
+// console.log(boilingPoint(temp2)) 350 is above boiling point
+// console.log(boilingPoint(temp3)) 212 is at boiling point
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
-const myNumbers1 = [3, 7, 0, 36, -9]
-const myNumbers2 = [8, -7, 42, 9, 13]
+// For this problem I'm going to use the .conCat() to combine both variables, then use .length to find the final index
+// I'll create a variable to group the first two variables together so I can utilize .concat()
+// Then I'll console.log my own variable to get the concatanation of both variable.
+// Lastly I'll .length the variable I created to receive an output of 10.
 
+// const myNumbers1 = [3, 7, 0, 36, -9]
+// const myNumbers2 = [8, -7, 42, 9, 13]
+//
+// const bothNums = myNumbers1.concat(myNumbers2)
+// console.log(bothNums) [3,  7,  0, 36, -9, 8, -7, 42,  9, 13]
+// console.log(bothNums.length) 10
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ovarB"
 
-const currentCohort = "Bravo 2022"
+// For this one I'll be using the array mutator .reverse.
+// All I have to do is console.log(currentCohort.reverse()) and I'll receive the output "1202 ovarB"
 
+// const currentCohort = "Bravo 2022"
+// console.log(currentCohort.reverse());
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
+// This one I'm not too sure what to do so I'm going to refer to my notes and instructor notes.
+// I'm completely lost.
 
-const myArray = [13, 34, 5, 10, 27, 42]
-
+// const myArray = [13, 34, 5, 10, 27, 42]
+//
+// if(myArray = 13, 5, 27,){
+//   console.log(myArray === Odd)
+// }
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
+// I had some idea what to do. I assume this is supposed to be a function, but I'm not sure how to type it out.
```

Comment:
Yes, you will need a function, just remember to break down the problem into small pieces.  For instance:
- Write a function that takes in 2 numbers
- Check which number is larger using a conditional statement
- If num1 is greater, subtract num2 from num1
- if num2 is greater, subtract num1 from num2